### PR TITLE
RK-16201 - update demos and project rooks on release

### PR DESCRIPTION
**Pull request by alexeygutkin**

### CODE REVIEW

**Comment by alexeygutkin** [comment]
Github Enforcer opened Task: [RK-16201](https://rookout.atlassian.net/browse/RK-16201)

**Review by ElDuderinos — APPROVED** [review]

File: .github/workflows/update-demo-rooks.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          cd src/checkoutservice
+          go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          go mod tidy
+          cd ../../src/frontend && go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          go mod tidy
+          cd ../../src/productcatalogservice && go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          go mod tidy
+          cd ../../src/shippingservice && go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          go mod tidy
+          git config --global user.email "sonario@rookout.com"
```
Comment (Urook) [inline .github/workflows/update-demo-rooks.yaml:54]: @alexeygutkin Shouldn't you do a cd .. && cd .. && cd .. && cd .. ?

Comment (alexeygutkin) [reply .github/workflows/update-demo-rooks.yaml:54]: @Urook not necessary

Comment (Urook) [reply .github/workflows/update-demo-rooks.yaml:54]: ok

**Pull request merged**